### DISCUSSION TITLE
Errror

### DISCUSSION THREAD

**Discussion by SMILESHx** [projectdiscovery/httpx#1945] [General]
root@Lenovo-PC:~/reports/# cat gau.txt |httpx -rl 1 -mc 401,403,405
Usage: httpx [OPTIONS] URL

Error: No such option: -r how to fix this

**Comment by ehsandeep** [comment]
http://nux.gg/wrong-httpx

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
